### PR TITLE
[BREAK] VideoConference

### PR DESCRIPTION
**Pull request by dougfabris**
## Proposed changes (including videos or screenshots)

In this PR we're deprecating the Video Conference functionality from the core of the application and introducing a **new video conference flow**:

Now the video conference feature will be agnostic so you'll be able to set the provider such as **Jisti** and **BBB** as apps from our marketplace: 

Video conferences settings are now global, allowing you to set the default provider

### [Enterprise Features]
- Video Conferences List

- Ringing function for direct messages

### CODE REVIEW

**Comment by Loriendaize** [comment]
I know that this message will be a surprise because we did not know each other, but the grace of God directed me to you and I would like you to read my message carefully.  Anyway my name is jhorn loriendaize
  , of Spanish origin and I live in Italy.  I suffer from a serious disease which condemns me to certain death, it is throat cancer.  I have a sum of €201,008 that I would like to donate to someone trustworthy and honest to put it to good use.  I have a used car business and I lost my husband 3 years ago which affected me a lot and I couldn't remarry until now.  We didn't have any children.  I would like to donate this sum before my death.  My days are numbered for the cause of this disease for which there is no cure.
  contact me by email: wj24011887@gmail.com

File: apps/meteor/client/providers/VideoConfProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+					setModal(
+						<GenericModal
+							variant='warning'
+							title='Action blocked'
+							confirmText={'Open_again'}
+							onConfirm={open}
+							onCancel={(): void => setModal()}
+							onClose={(): void => setModal()}
+						>{`Your browser has blocked the page {PAGE}. <br/> Consider enable popups for this domain url`}</GenericModal>,
```
Comment (pierre-lehnen-rc) [inline apps/meteor/client/providers/VideoConfProvider.tsx:33]: The content of this modal is still pending

Comment (dougfabris) [reply apps/meteor/client/providers/VideoConfProvider.tsx]: @pierre-lehnen-rc fixed!

**Review by ggazzo — APPROVED** [review]

**Review by thassiov — DISMISSED** [review]

**Review by pierre-lehnen-rc — DISMISSED** [review]

**Comment by ggazzo** [comment]
Ta muito ruim

**Review by d-gubert — DISMISSED** [review]

**Review by d-gubert — APPROVED** [review]

**Review by ggazzo — APPROVED** [review]

**Review by pierre-lehnen-rc — APPROVED** [review]

**Pull request merged**